asv.conf.json

```
@@ -15,8 +15,8 @@
 
     // Install using default qt install
     "build_command": ["python -V"],  // skip build stage
-    "install_command": ["in-dir={env_dir} python -m pip install {conf_dir}[all]"],
-    "uninstall_command": ["in-dir={env_dir} python -m pip uninstall -y {project}"],
+    "install_command": ["in-dir={env_dir} python -m pip install {conf_dir}[all] zarr"],
+    "uninstall_command": ["in-dir={env_dir} python -m pip uninstall -y {project} zarr"],
```

Optional: how about just including the extras/require for testing using the comma separated list of extras?

```suggestion
    "install_command": ["in-dir={env_dir} python -m pip install {conf_dir}[all,testing]"],
    "uninstall_command": ["in-dir={env_dir} python -m pip uninstall -y {project}"],
```

Maybe it's overkill, but in general I think it's reasonable to assume that benchmarking will need similar libraries to testing. Not sure about the change to `uninstall_command` or how important that action is since I guess we currently install more packages than we uninstall.

Alternatively, I guess we could have yet another section in `extras_require` specifically for benchmarks, but that's a little more maintenance burden.

---

Ha ! Yes, good point.